### PR TITLE
Parrot rename

### PR DESCRIPTION
**Pull request by lantiga**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,11 +59,26 @@ Avoiding code duplication is **not** a goal. **Readability** and **hackability**
...
 ```
 
-install dependencies
+Lit-Parrot currently relies on FlashAttention from PyTorch nightly. Until PyTorch 2.1 is released you'll need to install nightly manually.
+Luckily that is straightforward:
+
+**On CUDA**
+
+```bash
+pip install --index-url <https://download.pytorch.org/whl/nightly/cu118> --pre torch>=2.1.0dev
```
Comment (carmocca) [inline README.md:72]: Why the <>?

**Review by carmocca — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (lantiga) [reply README.md:72]: whoops, copy-paste artifact

**Review by awaelchli — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (carmocca) [reply README.md:72]:
```suggestion
pip install --index-url https://download.pytorch.org/whl/nightly/cu118 --pre torch>=2.1.0dev
```

File: README.md [hunk 2]
```diff
@@ -59,11 +59,26 @@ Avoiding code duplication is **not** a goal. **Readability** and **hackability**
...
+```
+
+**On CPU (incl Macs)**
+
+```bash
+pip install --index-url <https://download.pytorch.org/whl/nightly/cpu> --pre torch>=2.1.0dev
```
Comment (carmocca) [inline README.md:78]:
```suggestion
pip install --index-url https://download.pytorch.org/whl/nightly/cpu --pre torch>=2.1.0dev
```

**Pull request merged**